rapidtide/correlate.py

```diff
@@ -367,38 +363,60 @@ def shorttermcorr_2D(
     )
 
 
-# from https://stackoverflow.com/questions/20491028/optimal-way-to-compute-pairwise-mutual-information-using-numpy/20505476#20505476
 def calc_MI(x, y, bins=50):
```

Comment:
This function doesn't seem to be used anywhere outside of tests.

Reply:
It's been superseded in rapidtide by a cross-mutual information function, but I'd like to keep it around, since it's vetted, and I can imagine using it in the future.